force-app/test/record-update.spec.js

```diff
@@ -108,7 +106,7 @@ describe('Record update test', () => {
         console.log('Click Save at the bottom of Details panel');
         const footer = await baseRecordForm.getFooter();
```

Comment:
@lizaiv77 should I reload baseRecordForm here? I noticed that's what we do in Java

Reply:
no need